### ISSUE TITLE
setPlusRequestInitializer

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/google-api-java-client#645]
```
External references, such as a standards document, or specification?


Java environments (e.g. Java 6, Android 2.3, App Engine, or All)?

All

Please describe the feature requested.

Here's your typical builder structure:

    Plus plus =
        new Plus.Builder(httpTransport, jsonFactory, null).setGoogleClientRequestInitializer(
            new PlusRequestInitializer(API_KEY)).build();

But it would be easier to use if it were:

    Plus plus =
        new Plus.Builder(httpTransport, jsonFactory, null).setPlusRequestInitializer(
            new PlusRequestInitializer(API_KEY)).build();

In other words, add a setPlusRequestInitializer that looks something like:

public Builder setPlusRequestInitializer(PlusRequestInitializer initializer) {
  return (Builder) setGoogleClientRequestInitializer(initializer);
}

Note that we'd still keep setGoogleClientRequestInitializer.
```

Original issue reported on code.google.com by `yan...@google.com` on 31 Oct 2012 at 4:10

**Comment by GoogleCodeExporter** [comment]
Original comment by `rmis...@google.com` on 31 Oct 2012 at 5:21
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rmis...@google.com` on 31 Oct 2012 at 5:26
- Changed state: **Fixed**
